Runtime/UIElements/VisualElementExtension.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Reflection;
+using UnityEngine.UIElements;
+
+namespace StansAssets.Foundation.UIElements
+{
+    /// <summary>
+    /// Public representation of internal VisualElement API.
+    /// </summary>
+    public static class VisualElementExtension
+    {
+        [Flags]
```

Comment:
This is public so let's use proper UXML comments, as well as class description.

Something like:
Public representation of unity `PseudoStates` internal class.
to be consistent with:
https://api.stansassets.com/foundation/StansAssets.Foundation.UIElements.UIElementsUtility.html

And I have a feeling that we probably would want to use `PseudoStates` elsewhere so maybe it should be moved to the separate file since it's not bound to `VisualElementExtension` only

Reply:
`PseudoStates` is nested enum of a `VisualElement ` class but our public version is going to be used outside to call PseudoState related utility methods. @alexey-yaremenko what do you think?

Reply:
I think, that you should move PseudoStates enum class outside of VisualElementExtension.
1. we prob might use that in other places, just like @stan-osipov said.
2. it would be much more convenient if that's in a separate file. have two examples for you:
![image](https://user-images.githubusercontent.com/1619978/79048070-326e2400-7c23-11ea-87e6-dfdb06f93c19.png)
this one is the case where I moved PseudoStates out from VisualElementExtension.

![image](https://user-images.githubusercontent.com/1619978/79048092-47e34e00-7c23-11ea-93e0-25461a7a64ea.png)
and this is code from another project where PseudoStates is nested in VisualElementExtension.
it much more convenient even in terms of how code looks. it feels natively

p.s. and, please, don't forget about description for that enum